include/deal.II/grid/manifold.h

```diff
@@ -373,20 +436,17 @@ class Manifold : public Subscriptor
    */
   virtual
   Point<spacedim>
-  get_new_point (const std::vector<Point<spacedim> > &surrounding_points,
-                 const std::vector<double>           &weights) const;
+  get_new_point (const ArrayView<const Point<spacedim>> &surrounding_points,
+                 const ArrayView<const double>          &weights) const;
```

Comment:
Nice. That's going to make so many things simpler and more efficient.

Reply:
Yes. As I mentioned in the changelog, we pay a huge price for allocation that we can avoid with this breaking change :)